classes/monograph/PublishedMonographDAO.inc.php

```diff
@@ -17,6 +17,13 @@
 import('classes.monograph.PublishedMonograph');
 import('classes.monograph.MonographDAO');
 
+define('ORDERBY_DATE_PUBLISHED_ASC', 'datePublishedASC');
+define('ORDERBY_DATE_PUBLISHED_DESC', 'datePublishedDESC');
+define('ORDERBY_TITLE_ASC', 'titleASC');
+define('ORDERBY_TITLE_DESC', 'titleDESC');
+define('ORDERBY_SERIES_POSITION_ASC', 'seriesPositionASC');
+define('ORDERBY_SERIES_POSITION_DESC', 'seriesPositionDESC');
```

Comment:
Maybe to ask @NateWr because then two drop-down-boxes would be needed, right? S. https://github.com/pkp/pkp-lib/issues/744#issuecomment-171974532